hydromt/flw.py

```diff
@@ -270,7 +271,9 @@ def reproject_hydrography_like(
     da_elv: xarray.DataArray
         DataArray with elevation on destination grid.
     river_upa: float, optional
-        Minimum upstream area threshold for inflowing rivers, by default 5 [km2]
+        Minimum upstream area threshold [km2] for inflowing rivers, by default 5 km2
+    river_len: float, optional
```

Comment:
Went a bit deeper into the function this time and very nice one! 
It's quite complex and a lot of good things and exceptions are considered but a bit tough to understand what happens by just reading the docstring explnantion (maybe a bit more step by step?). The new addition of river_len is missing from the explanation.

Reply:
will add